core/templates/pages/contributor-dashboard-page/modal-templates/question-suggestion-review-modal.controller.ts

```diff
@@ -96,6 +96,7 @@ angular.module('oppia').controller('QuestionSuggestionReviewModalController', [
         const threadMessageBackendDicts = response.messages;
         $scope.reviewMessage = threadMessageBackendDicts.map(
           m => ThreadMessage.createFromBackendDict(m))[1].text;
+        $rootScope.$applyAsync();
```

Comment:
I suspected this was the issue, but what does this line do exactly? Trigger a re-render for async updates somehow?

/cc @bhavukJain

Reply:
It lets AngularJS know to refresh the view explicitly (more details in the [wiki](https://github.com/oppia/oppia/wiki/Angular-Migration#why-do-we-need-rootscopeapplyasync-with-http-requests).)